remotes/service.go

```diff
@@ -0,0 +1,55 @@
+package remotes
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb/v2"
+	"github.com/influxdata/influxdb/v2/kit/platform"
+	"github.com/influxdata/influxdb/v2/kit/platform/errors"
+)
+
+var (
```

Comment:
Formatting nitpick...the spacing here looks pretty gnarly. Would it be better to move the interface assertion out of the `var` block with the error for formatting purposes?